CustomApps/new-releases/index.js

```diff
@@ -273,21 +284,33 @@ function metaFromTrack(artist, track) {
                 name: artist.name,
                 uri: artist.link,
             },
-            imageURL: track.cover.uri,
+            imageURL: track.coverArt.sources[2].url,
             time,
-            trackCount: track.track_count,
+            trackCount: track.tracks.totalCount,
         };
     }
     return null;
 }
 
+var count = (function () {
+    var counter = 0;
+    return function (reset = false) {
+        return reset ? (counter = 0) : counter++;
+    };
+})();
+
```

Comment:
Left this as is... If it works, don't change it right?

But now it will just display error with that counter in console anyway... and for that there is `console.count()` iirc...
I'll leave it for now - if needs to be changed - just let me know :D